articles/deliverability/dedicated-ip-pools.md

```diff
@@ -17,7 +17,7 @@ You can provision new dedicated IPs directly to your Account via the UI.
 3. Once finished, click the "Confirm Purchases" button to purchase your dedicated IPs.
 4. The Billing page will then reflect your new purchase and your monthly invoice will be updated to reflect the charge for the dedicated IP(s). The dedicated IP(s) charge will be prorated by month.
 
-****NOTE: Each Account is limited to a maximum of 4 dedicated IPs at any one time***                                                                          
+****NOTE: Each SparkPost Account is limited to a maximum of 4 dedicated IPs at any one time. Enterpise Accounts have no limit on maximum dedicated IPs.***                                                                          
```

Comment:
Typo: `s/Enterpise/Enterprise/`

Reply:
Could we split this into a **Note** of its own?

Reply:
`...have no limit on dedicated IPs` - remove the `maximum` in favour of the `limit`?